src/test/java/com/keeper/homepage/domain/member/application/MemberProfileServiceTest.java

```diff
@@ -52,4 +57,71 @@ void setUp() throws IOException {
       assertThat(findThumbnail).isEqualTo(findMember.getProfile().getThumbnail());
     }
   }
+
+  @Nested
+  @DisplayName("회원 이메일 변경 테스트")
+  class UpdateMemberProfileEmailAddressTest {
+
+    @BeforeEach
+    void setUp() throws IOException {
+      member = memberTestHelper.generate();
+      otherMember = memberTestHelper.generate();
+      memberId = member.getId();
+      randomEmail = memberTestHelper.generate()
+          .getProfile()
+          .getEmailAddress()
+          .get();
+    }
+
+    @Test
+    @DisplayName("이메일 중복 시 예외를 던져야 한다.")
+    public void 이메일_중복_시_예외를_던져야_한다() {
+      assertThrows(BusinessException.class,
+          () -> memberProfileService.checkDuplicateEmailAddress(member.getProfile()
+              .getEmailAddress()
+              .get()));
```

Comment:
```suggestion
      assertThrows(BusinessException.class,
          () -> memberProfileService.checkDuplicateEmailAddress(member.getProfile()
              .getEmailAddress()
              .get()), MEMBER_EMAIL_DUPLICATE.getMessage());
```
nit: 이렇게 에러 메세지에 대한 체크도 할 수 있는 걸로 압니다! 하나의 서비스 코드가 던지는 예외가 많다면 메세지까지도 체크해보는 게 좋은 것 같더라구요! 물론 checkDuplicateEmailAddress 메서드는 하나의 예외만 던지므로 상관 없겠습니다 ㅎㅎ 참고하시면 좋을 것 같아서 코멘트 남겨보았어요 😇

Reply:
와~ 이런 방법도 있었군요!! 정확히 명시할 수 있는게 좋은거 같습니다!! 수덩하고 다음에도 쓰도록 하겠습니다 👍👍